utils/docker-entrypoint.sh

```
@@ -0,0 +1,34 @@
+  export OM_MAP_PATH=${WORKSPACE}/map.json
+
+  echo "OM_MAP_PATH not set, using default value: ${OM_MAP_PATH}"
+fi
+
+if [ ! -f "${OM_MAP_PATH}" ]; then
+  echo '{"type": "FeatureCollection", "features": []}' > ${OM_MAP_PATH}
+  echo "Created empty map file at ${OM_MAP_PATH}"
+fi
+
+# Source ROS environment
+source /opt/ros/${ROS_DISTRO}/setup.bash
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)